packages/itwinui-css/src/tag/tag.scss

```diff
@@ -21,7 +20,6 @@
 }
 
 a.iui-tag-basic {
-  @include anchor.iui-anchor;
   @include mixins.iui-focus($offset: 1px);
```

Comment:
Is this needed? Or just letting it (`@include mixins.iui-focus(…)`) come from the `.iui-anchor` is sufficient?

Reply:
It may not be. Let me check.

Reply:
Doesn't seem like it was. Removed in this [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/73f14a143597afbe40d9972bdab2586317536b48)